docs/usage.md

```diff
@@ -60,10 +73,28 @@ uses the "env" format which generates a simple shell script that `export`s the r
 when necessary). You can `source` this file to set the variables.
 
 Don't worry about the `--for-output-dir` option yet - and about the fact that the directory does not exist - it has to
-do with the target path where we will mount the output directory during the build.
+do with the target path where we will mount the output directory [during the build](#build-the-container).
 
 See also `cachi2 generate-env --help`.
 
+### Inject project files
```

Comment:
Just curious. AFAIK, gomod does not make use of project files as of now, right? Are we adding this section here just to keep the consistency between package managers?

Reply:
Yeah, the workflow should be always be fetch-deps -> generate-env, inject-files. Otherwise the workflow breaks if the project manager ever starts using project files.

But mainly, the section is here because the gomod example is the one that explains things while the other example(s) are just abridged versions.